Bucketeer/Sources/Internal/Evaluation/EvaluationDaoImpl.swift

```diff
@@ -1,10 +1,12 @@
 final class EvaluationDaoImpl: EvaluationDao {
+
     private let db: SQLite
 
     init(db: SQLite) {
         self.db = db
     }
 
+    // noted: userId is unused, should we remove it?
```

Comment:
Is this a bug?
In Android and the JS SDK, we use the user id when inserting the data into DB.

Reference.
https://github.com/bucketeer-io/android-client-sdk/blob/main/bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/evaluation/db/EvaluationDaoImpl.kt#L36-L46

It's odd because the get function uses the `userId`.

https://github.com/bucketeer-io/ios-client-sdk/pull/19/files#diff-717d2e8aa1104c9c44aa34ba1c6c04cef7fdb43dc27afd31d34b3f9c534cb1ddR18

Reply:
Please check my comment here https://github.com/bucketeer-io/ios-client-sdk/pull/19#discussion_r1284336081